CHANGELOG.md

```diff
@@ -1,5 +1,55 @@
 # `chainweb-node` Changelog
 
+## 2.1 (2020-08-11)
+
+This version replaces all previous versions. Any prior version will stop working
+on **2020-08-13T00:00:00Z**. Node administrators must upgrade to this version
+before that date.
+
+This version will stop working on **2020-10-08T00:00:00Z**.
+
+*Forks:*
+
+This version includes the fork that adds 10 additional chains to Chainweb
+resulting in 20 chains in total. The fork will occur on
+
+*   Mainnet at block height 852,054, which is expected to happen around
+    2020-08-20 16:55:14 UTC.
+
+The mining API of chainweb-node will start serving work items for the new chains
+starting at above block heights for the respective network. The mempool and pact
+service APIs will accept requests already some time before the transition. POW
+difficulties will re-adjust within a few hours after the transition. During that
+time block rates may be slightly higher or lower than usual.
+
+*Possibly Breaking Changes:*
```

Comment:
I thought there were more forking changes than just 1128?

Reply:
The other forking changes were only for testnet or for the mainnet-20 chain transition that didn't yet trigger. Here are all 9 commits since the last release: https://github.com/kadena-io/chainweb-node/compare/2.0...master